### ISSUE TITLE
Broken Feed (Text, All starred, active actions)

### ISSUE THREAD

**Issue by thatscoteng** [TracksApp/tracks#1929]
In Tracks 2.3.0, when going to view-> feeds,
Select the text version of 'All starred, active actions' [/todos.txt?tag=starred&token=xyz]

Then get error:
"""
We're sorry, but something went wrong.

If you are the application owner check the logs for more information.
"""

**Comment by GregSutcliffe** [comment]
I see this too. Logs show:

```
I, [2015-09-22T14:16:26.416812 #412191]  INFO -- : Started GET "/todos.txt?tag=starred&token=a38a8f3195d77f45cdd5b0f1794592986b92567d" for 127.11.70.129 at 2015-09-22 14:16:26 -0400
I, [2015-09-22T14:16:26.420332 #412191]  INFO -- : Processing by TodosController#index as TEXT
I, [2015-09-22T14:16:26.420436 #412191]  INFO -- :   Parameters: {"tag"=>"starred", "token"=>"a38a8f3195d77f45cdd5b0f1794592986b92567d"}
E, [2015-09-22T14:16:27.174904 #412191] ERROR -- : PG::UndefinedTable: ERROR:  missing FROM-clause entry for table "taggings"
LINE 1: ... = 'deferred' OR todos.state = 'pending')))) AND (taggings.t...
                                                             ^
: SELECT "todos".* FROM "todos" INNER JOIN contexts c_hidden ON c_hidden.id = todos.context_id WHERE "todos"."user_id" = $1 AND "todos"."state" = 'active' AND (NOT(todos.state = 'project_hidden' OR (c_hidden.state = 'hidden' AND (todos.state = 'active' OR todos.state = 'deferred' OR todos.state = 'pending')))) AND (taggings.tag_id = 27)  ORDER BY todos.due IS NULL, todos.due ASC, todos.created_at ASC
I, [2015-09-22T14:16:27.177582 #412191]  INFO -- :   Rendered contexts/_context.text.erb (19.4ms)
I, [2015-09-22T14:16:27.177753 #412191]  INFO -- :   Rendered todos/index.text.erb (53.8ms)
I, [2015-09-22T14:16:27.178180 #412191]  INFO -- : Completed 500 Internal Server Error in 758ms (ActiveRecord: 19.0ms)
F, [2015-09-22T14:16:27.181305 #412191] FATAL -- : 
ActionView::Template::Error (PG::UndefinedTable: ERROR:  missing FROM-clause entry for table "taggings"
LINE 1: ... = 'deferred' OR todos.state = 'pending')))) AND (taggings.t...
                                                             ^
: SELECT "todos".* FROM "todos" INNER JOIN contexts c_hidden ON c_hidden.id = todos.context_id WHERE "todos"."user_id" = $1 AND "todos"."state" = 'active' AND (NOT(todos.state = 'project_hidden' OR (c_hidden.state = 'hidden' AND (todos.state = 'active' OR todos.state = 'deferred' OR todos.state = 'pending')))) AND (taggings.tag_id = 27)  ORDER BY todos.due IS NULL, todos.due ASC, todos.created_at ASC):
    1: <%
    2:   todos_in_context = not_done_todos.select { |t| t.context_id == context.id }
    3:   if todos_in_context.length > 0
    4: -%> <%= context.name.upcase %>:
    5: <%= render :partial => "todos/todo", :collection => todos_in_context -%>
  app/views/contexts/_context.text.erb:2:in `_app_views_contexts__context_text_erb___1387713159190549381_45198180'
  app/views/todos/index.text.erb:1:in `_app_views_todos_index_text_erb___154646659570458006_44398080'
  app/controllers/todos_controller.rb:48:in `block (2 levels) in index'
  app/controllers/todos_controller.rb:29:in `index'
```

**Comment by ZeiP** [comment]
I'm getting a different error now. Currently the problem is that the starred tag is apparently not created after starring something, but only after another new tag is added after starring a first action.

**Comment by ZeiP** [comment]
I fixed a bug related to the tag being missing. The error message in log was a bit different though, so feel free to open a new issue if this still occurs in the current VCS version.
